tests/src/Kernel/fixtures/markup_rendering_patterns/accordion.yml

```diff
@@ -24,17 +27,19 @@ accordion_with_bare_layout:
       'div.accordion-item div.accordion-collapse.collapse': 3
       'div.accordion-item div.accordion-collapse.collapse.show': 1
     equals:
-      'div.accordion-item:nth-child(1) button.accordion-button': 'Accordion Item #1'
-      'div.accordion-item:nth-child(2) button.accordion-button': 'Accordion Item #2'
-      'div.accordion-item:nth-child(3) button.accordion-button': 'Accordion Item #3'
-      'div.accordion-item:nth-child(1) div.accordion-body': 'Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium'
-      'div.accordion-item:nth-child(2) div.accordion-body': 'Advice on living, working or travelling in the EU, on visas and immigration for non-EU citizens, European culture. Plus some extra text to make it a little bit longer.'
-      'div.accordion-item:nth-child(3) div.accordion-body': 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.'
+      'h4.bcl-heading': 'The accordion title'
```

Comment:
We need to add a different scenario where we don't pass the title, and one where we pass only the title without title tag. You can reduce the number of slides in those scenario. The same should happen for all places where we added the title tag and/or the title.